deploy.sh

```diff
@@ -59,12 +60,18 @@ function pollForSpecificDeployment() {
         STATUS=$(echo "$RESPONSE" | jq -r '.deploymentInfo.status')
 
         echo -e "${ORANGE}Deployment in progress. Sleeping 15 seconds. (Try $((++deadlockCounter)))";
-        echo -e "Instance Overview: ${RED}Failed ($FAILED_COUNT), ${BLUE}In-Progress ($IN_PROGRESS_COUNT), ${RESET_TEXT}Skipped ($SKIPPED_COUNT), ${BLUE}Pending ($PENDING_COUNT), ${GREEN}Succeeded ($SUCCESS_COUNT)"
-        echo -e "Deployment Status: $STATUS"
 
-        if [ "$FAILED_COUNT" -gt 0 ]; then
-            echo -e "${RED}Failed instance detected (Failed count over zero)."
-            exit 1;
+        if [ "$OVERVIEW_CHECK" == "null" ]; then
```

Comment:
Nervous about this check because of the comparison to null. We might be able to just use default values for jq so we can have fallbacks to like `?` for the values.

That will break our numeric check, but then maybe we explicitly look for a question mark. Just nervous on a check that explicitly looks for null in bash as a string

Reply:
I can set it so that the default value returned from Failed if not found results in a "?" that we can use for the conditional. Updating now.